client/state/reader/follows/reducer.js

```diff
@@ -64,28 +66,48 @@ function updatePostSubscription( state, { payload, type } ) {
 	};
 }
 
-/**
- * Tracks all known list objects, indexed by list ID.
- *
- * @param  {Array} state  Current state
- * @param  {Object} action Action payload
- * @return {Array}        Updated state
- */
 export const items = createReducer( {}, {
-	[ READER_FOLLOW ]: ( state, action ) => {
+	[ READER_RECORD_FOLLOW ]: ( state, action ) => {
 		const urlKey = prepareComparableUrl( action.payload.url );
 		return {
 			...state,
 			[ urlKey ]: merge( {}, state[ urlKey ], { is_following: true } ),
 		};
 	},
-	[ READER_UNFOLLOW ]: ( state, action ) => {
+	[ READER_RECORD_UNFOLLOW ]: ( state, action ) => {
 		const urlKey = prepareComparableUrl( action.payload.url );
 		return {
 			...state,
 			[ urlKey ]: merge( {}, state[ urlKey ], { is_following: false } ),
 		};
 	},
+	[ READER_FOLLOW ]: ( state, action ) => {
+		const urlKey = prepareComparableUrl( action.payload.feedUrl );
+		return {
+			...state,
+			[ urlKey ]: merge(
+				{ feed_URL: action.payload.feedUrl },
```

Comment:
We just had an offline discussion about how if you follow from a feed-search it will populate with just ```
{ 
   [  url ] : { is_following: true }
}
```

This wouldn't give the ui enough to go off to fetch the site/feed in the meantime.  We are discussing also throwing in an optional siteId/feedId to the reducer/action creator